components/Common/NavBar.tsx

```diff
@@ -42,20 +42,20 @@ const NavBar: FC = () => {
   const router = useRouter();
   const { events } = useAppSelector((state) => state.events);
   const { blogs } = useAppSelector((state) => state.blogs);
-  const { partners } = useAppSelector((state) => state.partners);
+  const { sponsors } = useAppSelector((state) => state.sponsors);
   const { academics } = useAppSelector((state) => state.academics);
   const links = [
     { label: 'Events', href: '/Events' },
     { label: 'Blogs', href: '/Blogs' },
-    { label: 'Partners', href: '/Partners' },
```

Comment:
we still want a partners tab / page along with sponsors

Reply:
Partners does not currently exist in the backend CMS. should i hold off from editing until it is pushed?

Reply:
oh thats right mb we should remove it